support-frontend/assets/helpers/forms/checkouts.ts

```diff
@@ -228,12 +228,28 @@ function getPaymentDescription(
 	return '';
 }
 
-const simpleFormatAmount = (
-	currency: Currency,
-	amount: number | string,
-): string => {
+function round(amount: number) {
+	/**
+	 * This rounds a `number` to the second decimal.
+	 *
+	 * `Number.toFixed` returns a string which is not useful for calculations
+	 * and would need unnecessary type conversions
+	 */
+	return Math.round(amount * 1e2) / 1e2;
+}
+
+const simpleFormatAmount = (currency: Currency, amount: number): string => {
```

Comment:
And this is the reading type change I wanted to just be `number`.

The method itself is refactored to do rounding. Happy to remove this as that was superfluous to this refactor.